### DISCUSSION TITLE
https redirect without custom server

### DISCUSSION THREAD

**Discussion by jasonfarrell** [vercel/next.js#15191] [Help]
I'm using `next start` to take advantage of the Automatic Static Optimization, rather than using a custom node server. I can't find for the life of me how to redirect our production routes to https on the backend.

Is there some configuration that will force https without creating a custom server?

**Comment by MatthiasEg** [comment]
@jasonfarrell I'm also interested in what you were trying to do. Did you find a solution in the meantime?

**Reply by jasonfarrell** [reply]
@MatthiasEg never found what I wanted, which was a config in next to redirect to https on the backend. 

Here's what I found the options to be:

1) add a client side redirect, but users will have to wait for the insecure page to load and then it redirects
2) use vercel as the host
3) use cloudflare to redirect dns https://support.cloudflare.com/hc/en-us/articles/204144518-SSL-FAQ#h_a61bfdef-08dd-40f8-8888-7edd8e40d156 — with this though, you'll have to move your DNS (or client's DNS) to cloudflare, which is not really a good option
